src/extension/modules/TxDetails.tsx

```diff
@@ -3,23 +3,29 @@ import { useTranslation } from "react-i18next"
 import { useNetwork } from "data/wallet"
 import { Grid } from "components/layout"
 import { Dl, ToNow } from "components/display"
-import { ReadMultiple } from "components/token"
+import { Read } from "components/token"
 import { getIsNativeMsgFromExternal, TxRequest } from "../utils"
 import Message from "./Message"
 import styles from "./TxDetails.module.scss"
+import { useNativeDenoms } from "data/token"
 
 const TxDetails = ({ origin, timestamp, tx }: TxRequest) => {
   const { msgs, memo, fee, chainID } = tx
 
   const { t } = useTranslation()
   const network = useNetwork()
+  const readNativeDenom = useNativeDenoms()
+  const { decimals } = readNativeDenom(network[chainID].baseAsset)
 
   const fees = fee?.amount.toData()
   const contents = [
     { title: t("Network"), content: `${network[chainID]?.name} (${chainID})` },
     { title: t("Origin"), content: origin },
     { title: t("Timestamp"), content: <ToNow update>{timestamp}</ToNow> },
-    { title: t("Fee"), content: fees && <ReadMultiple list={fees} /> },
+    {
+      title: t("Fee"),
+      content: fees && <Read {...fees[0]} decimals={decimals} />,
```

Comment:
Does this `Read` component need to be refactored to take into account `St 691 decimals 3` PR?

Reply:
Its not currently causing any problems, and I think they'll both be replaced going forward by station-ui, so no reason to alter for now.

Reply:
I think in the `St 691 decimals 3` PR, `decimals` are evaluated in the actual component utilizing `denom`:
https://github.com/terra-money/station/pull/488/files#diff-9b8eef0f962d8ce82c779e713561a144fa29d46cadd49cb60f72ee07d40f2657R27-R31

So I think `denom` needs to be passed to the `Read` component here instead of `decimals`.

Otherwise, make the change in your `St 691 decimals 3` PR.

Let me know if I'm missing something.

Reply:
this is extension vs. station